api.wordpress.org/public_html/dotorg/github/activity.php

```diff
@@ -71,6 +71,22 @@ function github_user_to_user_id( $user ) {
 	return $user_id ? intval( $user_id ) : false;
 }
 
+/**
+ * Save github activity to database.
+ *
+ * @param array $data
+ *
+ * @uses apply_filters() Calls 'wporg_github_added_activity'
+ */
+function save_github_activity( $data ) {
+	$wpdb->insert(
+		'wporg_github_activity',
+		$data
+	);
+
+	apply_filters( 'wporg_github_added_activity', $data );
```

Comment:
```suggestion
	do_action( 'wporg_github_added_activity', $data );
```

I think an action fits better here, since the value isn't being changed.

Reply:
I understand the difference now :) Thanks!